app/bundles/EmailBundle/Controller/AjaxController.php

```diff
@@ -317,4 +318,27 @@ public function heatmapAction(Request $request, EmailModel $model): JsonResponse
             'legendTemplate'    => $legendTemplate,
         ]);
     }
+
+    public function getEmailUsagesAction(Request $request, EmailDependencies $emailDependencies): JsonResponse
+    {
+        $emailId = (int) $request->query->get('id');
+
+        if (0 === $emailId) {
+            return $this->sendJsonResponse([
+                'success' => 0,
```

Comment:
I'd suggest to stop using the `success` flag in the response and use HTTP statuses instead. I know it's used elsewhere in Mautic but it's a bad habit.

Reply:
You're right, I changed it.